tests/unit/test_sandbox.py

```diff
@@ -294,6 +294,52 @@ def test_sandbox_jupyter_plugin(temp_dir):
             )
             box.close()
 
+def _agentskills_test(box):
```

Comment:
Maybe rename this as `_test_sandbox_jupyter_agentskills_fileop_pwd_impl` to be more concise? Since it is the core logic reused by the other two tests.

Reply:
Question: that new test does not require or cause the images to be actually downloaded, right?

Reply:
Fixed it

Reply:
> Question: that new test does not require or cause the images to be actually downloaded, right?问题：新测试不需要或导致实际下载图像，对吧？

sorry, I don't understand your meaning. Are you meaning the `test_agnostic_sandbox_jupyter_agentskills_fileop_pwd` unittest whether need to download custom sandbox base image?

Reply:
Correct, that is what I meant.

Reply:
@tobitege 
Yes, this will result in downloading the image. We need to test whether sandbox container can work finely in different custom image

Reply:
This, imho, should not happen in a unit test. This test sould be written as an integation test then.

Reply:
> This, imho, should not happen in a unit test

sandbox tests aren't unit tests. They are just placed in a weird place.

Reply:
Well, I called it what it is packaged as right now. 🤣

Reply:
huh.. should I remove the test in the file?

Reply:
> huh.. should I remove the test in the file?

I'd defer to @li-boxuan 's opinion, I might be wrong with my previous post.